kafka-ui-e2e-checks/src/test/java/com/provectus/kafka/ui/tests/ConnectorsTests.java

```diff
@@ -71,7 +74,7 @@ void updateConnector() {
     @SneakyThrows
     @DisplayName("should delete connector")
     @Test
-    @Disabled
+    @Disabled("disable test due 500 error during create connector via api")
```

Comment:
we can revert these, these tests are being fixed within #1314 #1480

Reply:
Ok